### ISSUE TITLE
For users inferred as low quality or marked as low quality manual, do we include them in our API

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2967]
Chu and I were talking today. She is doing an analysis of our Seattle data. One question that came up is whether our APIs return data from "low quality" users (that is, users that our algorithm inferred as low quality or that were marked as manual). We've been recently shifting our visualizations and how we compute "city completion" by ignoring data from low quality users, so I was curious how this is being handled with our APIs. Perhaps we could have a flag in the API, something like `includeLowQualityData=true`

Related to https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2966 and https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2952.

**Comment by misaugstad** [comment]
We are not currently including that low quality data. Adding a flag for that is a bit more complicated than it initially seems, because we need to decide whether or not to include the low quality data early in the clustering process. When we do nightly clustering, we could potentially run the second stage of clustering (multi-user clustering) twice (once with and once without low quality data) and store both of those in the database in order to support adding that flag.

**Comment by jonfroehlich** [comment]
Gotcha. Useful to know regardless.

We should include such information in the API docs themselves (but this goes back to generally improving those docs anyway https://github.com/ProjectSidewalk/SidewalkWebpage/issues/2754)

**Comment by misaugstad** [comment]
>I believe this is now addressed in our APIs via a flag, right @misaugstad? 

Not really! We have a flag in the /v3/api/userStats API, but this hasn't been addressed in any of our other API endpoints.

Here are the points that I think we need to decide on or otherwise resolve:
1. Do we want to provide a version of the clustered data without filtering? I don't think that we do. It would mean running clustering two separate times each night and storing the output of both. I don't see many people wanting this, and it's probably more likely that they'd want clustering rerun with a unique set of filters, which we aren't going to do.
2. In our documentation for the /labelClusters API, we should mention this filtering that we're doing, probably with a description of our algorithm for determining a user with low quality data
3. If someone wanted to get the raw labels from high quality users themselves, they would have to join the /rawLabels API with the /userStats API... We should just add a flag to the /rawLabels API to make this easier (and more clear)

**Comment by jonfroehlich** [comment]
Very good points. 

1. Agree with you.
2. Yes, important! We might want to have this as a programatically stored statement since we need to use it in multiple places (e.g., on the rawLabels page as well)
3. Right
